paper.md

```diff
@@ -67,25 +67,27 @@ affiliations:
-The clear, modular design of `ros_control` makes it ideal for both research and industrial use and has indeed seen many such applications to date. `ros_control` is out-of-the-box compatible with 3rd party software such as `MoveIt!` [@chitta2012moveit],  the `ROS navigation stack`, `Gazebo`[@koenig2004design] and others.
+In recent years the Robotics Operating System [@quigley2009ros] (ROS) has become the 'de facto' standard framework for robotics software development. The `ros_control` framework provides the capability to implement and manage robot controllers with a focus on both _real-time performance_ and _sharing of controllers_ in a robot-agnostic way. The primary motivation for a sepate robot-control framework is the lack of realtime-safe communication layer in ROS. Furthermore, the framework implements solutions for controller-lifecycle and hardware resource management as well as abstractions on hardware interfaces with minimal assumptions on hardware or operating system. The clear, modular design of `ros_control` makes it ideal for both research and industrial use and has indeed seen many such applications to date. The idea of `ros_control` originates from the `pr2_controller_manager` framework specific to the PR2 robot but `ros_control` is fully robot-agnostic. Controllers _expose standard ROS interfaces_ for out-of-the box 3rd party solutions to robotics problems like manipulation path planning (`MoveIt!` [@chitta2012moveit]) and autonomous navigation (the `ROS navigation stack`). `ros_control` also provides several libraries to support writing custom controllers.
+<!-- with some ideas borrowed from OROCOS [@bruyninckx2001open].  -->
 
-The backbone of the framework is the Hardware Abstraction Layer, which serves as a bridge to different simulated and real robots. This abstraction is provided by the `hardware_interface::RobotHW` class; specific robot implementations have to inherit from this class.  Instances of this class are then used to interface the robot hardware (including some low-level sensors) to higher-level controllers. This allows higher-level controllers to be hardware-agnostic and easily shareable.
+The backbone of the framework is the Hardware Abstraction Layer, which serves as a bridge to different simulated and real robots. This abstraction is provided by the `hardware_interface::RobotHW` class; specific robot implementations have to inherit from this class. Instances of this class model hardware resources provided by the robot such as electric and hydraulic actuators and low-level sensors such as encoders and force/torque sensors. It also allows for mix and match heterogeneous hardware or swap out components transparently whether it is a real or simulated robot. Controllers are also hardware-agnostic. They specify required hardware resources to be able to operate. Their lifecycle is not static, and controller configurations can be queried/modified at runtime: start/stop, reconfigure.
```

Comment:
Thanks to this layer, controllers can be hardware-agnostic and easily sharable among different robots. 

I would remove "They specify required hardware resources to be able to operate." or rephrase it by talking about the hardware interfaces (could be useful to explain the image)

"Their lifecycle is not static, and controller configurations can be queried/modified at runtime: start/stop, reconfigure." should be moved in the controller_manager part